source/common/stream_info/filter_state_impl.cc

```diff
@@ -31,5 +31,32 @@ const FilterState::Object* FilterStateImpl::getDataGeneric(absl::string_view dat
   return it->second.get();
 }
 
+void FilterStateImpl::addToListGeneric(absl::string_view list_name,
+                                       std::unique_ptr<Object>&& item) {
+  // TODO(Google): Remove string conversion when fixed internally.
+  const std::string name(list_name);
+  if (list_storage_.find(name) == list_storage_.end()) {
+    list_storage_[name] = std::vector<std::unique_ptr<FilterState::Object>>();
```

Comment:
I think this can be `list_storage_[name] = {};`

Reply:
Yeah, not sure why the compiler is upset about that, but I'm fine with leaving this as you had it.